Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ruby:2.6.5
+RUN apt-get update -qq && apt-get install -qq curl postgresql-client supervisor
+
+# Install node and yarn
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update -qq && apt-get install -qq nodejs yarn
+
+COPY config/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+
+RUN mkdir /app
+WORKDIR /app
+
+# Install ruby deps
+COPY Gemfile .
+COPY Gemfile.lock .
+RUN bundle install
+
+# Install node deps
+COPY package.json .
+COPY yarn.lock .
+RUN yarn install
+
+COPY . .
+
+CMD ["/usr/bin/supervisord"]
```

Comment:
@rakshans1 can't we directly run `bundle exec rails s -b 0.0.0.0` instead of running it via supervisor? I believe that would be more straight forward and better aligned with Docker.

Reply:
@BilalBudhani I think front-end devs need webpack server for debugging that's why Rakshan used `supervisor`.

Reply:
@BilalBudhani For frontend live reloading we need to run webpack dev server. That why I added supervisor to run two processes.

Reply:
@shivamsupr I can see we are running two processes inside one Docker container (webpacker and rails) which is not the right way. We will need to add another service inside docker-compose to separately run webapcker process. Check https://github.com/rails/webpacker/blob/master/docs/docker.md

Reply:
@BilalBudhani I think this is the right approach. Thanks

Reply:
@BilalBudhani Agreed. Adding another service for webpacker